ISSUE_TEMPLATE

```diff
@@ -0,0 +1,28 @@
+## Informations
+<!--- Orchestra version (1.0, 1.1, master, ...) --->
+**Orchestra Version** : 
+
+**Issue Type**: 
+<!--- 
+Add x in type of your issue 
+For instance if it is an issue for a bug : 
+- [ x] Bug
+- [ ] Feature
+- [ ] Question
+- [ ] Other
+--->
+- [ ] Bug
+- [ ] Feature
+- [ ] Question
+- [ ] Other
+
+## Description
+<!--- 
+If is a bug issue
+Explain the issue with screenshot if possible and what do you expected
```

Comment:
I'll add ``Don't omit specific context such as brwoser version if related to a javascript/css bug for instance.``